pubspec.yaml

```diff
@@ -13,6 +13,7 @@ dependencies:
   collection: ^1.15.0
   logging: ^1.0.1
   meta: ^1.3.0
+  rohd_vf: ^0.4.1
```

Comment:
ROHD should not depend on ROHD-VF, since ROHD-VF depends on ROHD.  We can't have a cyclic dependency like that.  It's possible that the example code just won't be executable in this repo, or otherwise it would have to be an independent sub-project with it's own pubspec.yaml file